life_hc/MySQL/WantedSystem/fn_wantedFetch.sqf

```diff
@@ -11,39 +11,43 @@
     Description:
     Displays wanted list information sent from the server.
 */
-private ["_ret","_list","_result","_queryResult","_units","_inStatement"];
-_ret = [_this,0,objNull,[objNull]] call BIS_fnc_param;
+
+params [
+    ["_ret", objNullm [objNull]]
```

Comment:
Yep good spot, and thanks for testing 👍 It would be great if sometime in the near future, after the mres functions have been removed, you are able to test the HC part again.

Reply:
no Problem, just let me know. Our Testserver is now set up to support HC, so this should be the smallest problem ;)

Reply:
initHC: you forgot to add the SQL_Custom definition there... missed to point that out ;)

Reply:
Good spot again 👍